src/main/resources/db/migration/V4__add-currency-to-account.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE accounts ADD COLUMN currency VARCHAR(5);
```

Comment:
чого 5 а не три? У всіх рівно три буде: https://uk.wikipedia.org/wiki/%D0%9A%D0%BB%D0%B0%D1%81%D0%B8%D1%84%D1%96%D0%BA%D0%B0%D1%86%D1%96%D1%8F_%D0%B2%D0%B0%D0%BB%D1%8E%D1%82_(ISO_4217)#%D0%9F%D0%B5%D1%80%D0%B5%D0%BB%D1%96%D0%BA_%D0%B4%D1%96%D1%8E%D1%87%D0%B8%D1%85_%D0%BA%D0%BE%D0%B4%D1%96%D0%B2
```suggestion
ALTER TABLE accounts ADD COLUMN currency VARCHAR(3);
```

Reply:
вдруг какие-то скобки по бокам добавились-бы)))